### PR TITLE
Syangja Added

### PR DESCRIPTION
**Pull request by ori-on1**
## What kind of change does this PR introduce?

- [x] Bug Fix
- [x] Documentation change
- [x] Added a new city
- [x] Other(s)

### CODE REVIEW

File: data/borderlines.json [hunk 1]
```diff
@@ -127,6 +127,47 @@
...
+      "currency": {
+        "name": "Nepalese Rupee",
+        "code": "NPR"
+      },
+      "landArea": {
+        "value": 1164,
+        "unit": "km²"
+      },
+      "elevation": {
+        "value": 1012,
```
Comment (thenishantsapkota) [inline data/borderlines.json:145]: This still isn't correct
When I searched for elevation I found this
image

**Review by thenishantsapkota — CHANGES_REQUESTED** [review]

**Review by ori-on1 — COMMENTED** [review]
Corrected !

File: data/borderlines.json [hunk 2]
```diff
@@ -127,6 +127,46 @@
...
+    {
+      "name": "Syangja",
+      "latitude": 28.0197,
+      "longitude": 83.8049,
+      "timezone": "Asia/Kathmandu",
+      "officialLanguage": "Nepali",
+      "currency": {
+        "name": "Nepalese Rupee",
+        "code": "NPR"
+      },      "landArea": {
```
Comment (thenishantsapkota) [inline data/borderlines.json:139]: The code isn't properly formatted.
Please format the code using prettier.

**Review by thenishantsapkota — CHANGES_REQUESTED** [review]

**Review by ori-on1 — COMMENTED** [review]
formatted correctly

**Review by thenishantsapkota — APPROVED** [review]
LGTM!
Thank you for your contribution @ori-on1!

**Comment by ori-on1** [comment]
Cool @thenishantsapkota thank you for your guidance and review.

**Pull request merged**